src/main/java/uk/gov/hmcts/reform/migration/service/DataMigrationServiceImpl.java

```diff
@@ -38,21 +40,13 @@ public Map<String, Object> rollback(Long id, Map<String, Object> data, String us
     public Map<String, Object> migrate(Long caseId, Map<String, Object> data, String userToken, String authToken) {
         if (data == null) {
             return null;
-        }
-
-        String channelChoice = "";
-        if ("No".equals(data.get("paperForm"))) {
-            channelChoice = "Digital";
-        } else {
+        } else if (null == data.get("applicationSubmittedDate")) {
             AuditEvent auditEvent = getAuditEvent(caseId, userToken, authToken);
```

Comment:
Add orElse for if the event is not found  like below
    private AuditEvent getAuditEvent(Long caseId, SecurityDTO securityDTO) {
        return auditEventService.getLatestAuditEventByName(caseId.toString(), createCaseFromBulkScan,
                securityDTO.getAuthorisation(), securityDTO.getServiceAuthorisation())
                .orElseThrow(() -> new IllegalStateException(String
                .format("Could not find any event other than %s event in audit", createCaseFromBulkScan)));
Otherwise it may end up in Nullpointer Exception if no event is found.       
    }